lib/gosquared/index.js

```diff
@@ -90,21 +91,62 @@ GoSquared.prototype.page = function(page){
 
 GoSquared.prototype.identify = function(identify){
   var traits = identify.traits({
-    userId: 'user_id',
-    created: 'created_at'
+    createdAt: 'created_at',
+    firstName: 'first_name',
+    lastName: 'last_name',
+    title: 'company_position',
+    industry: 'company_industry'
+  });
+
+  // https://beta.gosquared.com/docs/tracking/api/#properties
+  var special = [
+    'id',
+    'email',
+    'name',
+    'first_name',
+    'last_name',
+    'username',
+    'description',
+    'avatar',
+    'phone',
+    'created_at',
+    'company_name',
+    'company_size',
+    'company_position',
+    'company_industry'
+  ];
+
+  // Segment allows traits to all be in a flat object
+  // GoSquared requires all custom properties to be in a `custom` object,
+  // separate from special properties
+  // this isn't ideal, but otherwise all Segment users would need to use `custom`
+  var props = {};
+  var customObj = is.object(traits.custom);
+  if (customObj) props.custom = traits.custom;
+
+  each(traits, function(name, value){
+    // don't double move `custom` (but allows non-object custom)
+    if (name === 'custom' && customObj) return;
+
+    if (special.indexOf(name) !== -1) {
+      props[name] = traits[name];
+    } else {
+      if (!props.custom) props.custom = {};
+      if (!(name in props.custom)) props.custom[name] = traits[name];
+    }
   });
 
```

Comment:
Sadly using the little individual lodash npm packages doesn't work with duo, so we can use a few component implementations for each of the functions we need. Here are `keys`: [matthewp/keys](https://github.com/matthewp/keys), `pick`: [ianstormtaylor/pick](https://github.com/ianstormtaylor/pick), `difference`: [bredele/array-diff](https://github.com/bredele/array-diff), and `omit`: [harrietgrace/omit](https://github.com/harrietgrace/omit). You can add those to the component.json and it should be ready to go!